### PR TITLE
Issue#594 Adding tutorial to Chat Activity

### PR DESCRIPTION
**Pull request by Hemansh31**
@llaske everything except localiztion is working fine and is according to what you mentioned

### CODE REVIEW

**Comment by s-mag** [comment]
@Hemansh31 Nice but you should not have console messages. And for the localization see step5 of the development tutorial.

**Comment by Hemansh31** [comment]
@s-mag 
I will remove the console messages and as for the localization I did everything according to the tutorial but it wasn't working and I wasn't able to spot the bug, would be a great help if you could review my code

File: activities/ChatPrototype.activity/css/activity.css [hunk 1]
```diff
@@ -6,6 +6,10 @@ background-image:url(../activity/activity-icon.svg);
 background-image:url(../lib/sugar-web/graphics/icons/actions/zoom-home.svg);
 }
 
+#main-toolbar #help-button {
+  background-image: url(../icons/help.svg);
```
Comment (s-mag) [inline activities/ChatPrototype.activity/css/activity.css]: Fix Indentation

File: activities/ChatPrototype.activity/js/activity.js [hunk 2]
```diff
@@ -37,6 +37,8 @@ define(["sugar-web/activity/activity","webL10n","sugar-web/graphics/palette","su
         var messageContent = document.getElementById('content');
         var imageUpload = document.getElementById('image-upload');
 
+
```
Comment (s-mag) [inline activities/ChatPrototype.activity/js/activity.js]: No need to add space here

File: activities/Write.activity/lib/webL10n.js [hunk 3]
```diff
@@ -983,6 +983,7 @@ define(function (require) {
     get: function l10n_get(key, args, fallback) {
       var data = getL10nData(key, args) || fallback;
       if (data) {
+        console.log(data);
```
Comment (s-mag) [inline activities/Write.activity/lib/webL10n.js]: Remove this line

File: activities/ChatPrototype.activity/js/activity.js [hunk 4]
```diff
@@ -225,6 +227,9 @@ define(["sugar-web/activity/activity","webL10n","sugar-web/graphics/palette","su
               }
 
             }
+            document.getElementById("help-button").addEventListener('click', function(e) {
+				tutorial.start();
```
Comment (s-mag) [inline activities/ChatPrototype.activity/js/activity.js]: Fix Indentation here

**Comment by llaske** [comment]
Regarding localization, it seems that this activity use a non usual way to initialize localization. So the l10n strings are not yet initialized when the tutorial object is define. Don't bother with it. I will fix it in another issue.

BTW, miss a title bar in the message popup:

image

Plus you should add popups for image button (appears only when the activity is shared) and for emojis buttons.

**Comment by Hemansh31** [comment]
@llaske I am currently making the changes you asked for but since I wanted to submit a PR for the ChessBoard Task I closed for a while. Will reopen once I am done with the changes

**Pull request closed without merging**